### ISSUE TITLE
Lab 1, Ex 3, Step 4 - Command not work

### ISSUE THREAD

**Issue by SystemOutCoding** [MicrosoftLearning/AZ-203-DevelopingSolutionsforMicrosoftAzure#22]
Module: AZ-203
Lab/Demo: 01
Task: 3
Step: 4

4.  Use the following command to save the name of the most recently created **container registry** in a Bash shell variable *acrName*:

    ```
    $acrName=(az acr list --query "max_by([], &creationDate).name" --output tsv)
    ```

It's not working within Azire Cloud Shell

```
// I gonna try
junhyun@Azure:~/clouddrive$ az acr list --query "max_by([], &creationDate).name" --output tsv
>>az203bjh201906
junhyun@Azure:~/clouddrive$ $acrName=(az acr list --query "max_by([], &creationDate).name" --output tsv)
>> bash: syntax error near unexpected token `az'
// It's not working

junhyun@Azure:~/clouddrive$ acrName=`(az acr list --query "max_by([], &creationDate).name" --output tsv)`
junhyun@Azure:~/clouddrive$ echo $acrName
>>az203bjh201906
// Done
```

**Comment by willnix86** [comment]
If you throw a $ in-front of (az acr...) it works too.

**Comment by seesharprun** [comment]
Resolved in [Version 20190701.04](https://github.com/MicrosoftLearning/AZ-203-DevelopingSolutionsforMicrosoftAzure/releases/tag/v20190701.04). The code that follows best practices was:

```
acrName=$(az acr list --query "max_by([], &creationDate).name" --output tsv)
```
